### DISCUSSION TITLE
Error Build Whitelabel Xcode iOS App

### DISCUSSION THREAD

**Discussion by firmanMMSGI** [RocketChat/Rocket.Chat.ReactNative#3662] [Q&A] [answered]
Hello warm regards, anyone can. helped me, i found some simple things while doing build for ios whitelabel, initially download and i build straight away without making any changes, attached error
<img width="1440" alt="Screen Shot 2022-01-28 at 14 06 38" src="https://user-images.githubusercontent.com/94961982/151503151-ad52e858-1778-43fc-bbe3-5cc877744ad2.png">

**Comment by diegolmello** [comment, marked as answer]
Have you followed this? https://github.com/RocketChat/Rocket.Chat.ReactNative/blob/develop/CONTRIBUTING.md#how-to-run
